domains/abdullah.json

```
@@ -1,9 +1,11 @@
 {
-    "owner": {
-        "username": "mrpsycho-dev",
-        "email": "stinkmang6@gmail.com"
-    },
-    "records": {
-        "CNAME": "mrpsycho-dev.github.io"
-    }
+  "owner": {
+    "username": "mrpsycho-dev",
+    "email": "stinkmang6@gmail.com"
+  },
+  "records": {
+    "CNAME": "mrpsycho-dev.github.io",
+    "TXT": "_github-pages-challenge-mrpsycho-dev.abdullah.is-a.dev=855cd3278a7c54ebaed66fe5e7e33a"
```

read and follow the guide -> https://docs.is-a.dev/guides/github-pages/#verifying-your-is-adev-subdomain-with-github-pages
```suggestion
    "CNAME": "mrpsycho-dev.github.io"
```